uint8 id
--- 
int16 status
string message
string hardware_id
